test/test_ngraph_tensor_manager.cpp

```diff
@@ -202,12 +245,24 @@ TEST_F(NGraphTensorManagerTest, NoVariablesHasPrefetch) {
   int number_of_outputs = 2;
 
   // expected
+  // var
   vector<int> empty;
+  vector<int> expected_out_indexes_need_copy = FillRange(number_of_outputs);
```

Comment:
The test might seem redundant. I have 4 states capturing the different combinations for existence of Variable and Prefetch. I can reduce it 2 states.